openvdb/math/Stencils.h

```diff
@@ -34,32 +37,33 @@ template<typename DerivedType, typename GridT, bool IsSafe>
 class BaseStencil
 {
 public:
-    typedef GridT                                       GridType;
-    typedef typename GridT::TreeType                    TreeType;
-    typedef typename GridT::ValueType                   ValueType;
-    typedef tree::ValueAccessor<const TreeType, IsSafe> AccessorType;
-    typedef std::vector<ValueType>                      BufferType;
-    typedef typename BufferType::iterator               IterType;
+    using GridType = GridT;
+    using TreeType = typename GridT::TreeType;
+    using ValueType = typename GridT::ValueType;
+    using AccessorType = tree::ValueAccessor<const TreeType, IsSafe>;
+    // Use boost::container::vector instead of std::vector,
+    // because std::vector<bool> doesn't provide a data() method.
+    using BufferType = boost::container::vector<ValueType>;
```

Comment:
@kmuseth - this is first of two new boost dependencies being introduced in this PR.